ruskeyverbs/models.py

```diff
@@ -26,6 +29,42 @@ class Verb(models.Model):
     def __str__(self):
         return self.infinitive
 
+    def get_earliest_due_date(self, my_user):
+        examples = self.example_set.all()
+        date_list = []
+        future_date = datetime.date.today() + datetime.timedelta(weeks=9999)
+        for my_example in examples:
+            try:
+                my_due_date = my_example.performanceperexample_set.filter(user=my_user)[0].due_date
+                date_list.append(my_due_date)
+            except ObjectDoesNotExist:
+                date_list.append(future_date)
+            except IndexError:
+                date_list.append(future_date)
+        if date_list:
+            return min(date_list)
+
+    def is_overdue(self, my_user):
+        return self.get_earliest_due_date(my_user) < datetime.date.today()
+
+    def due_date_for_display(self, my_user):
```

Comment:
This method appears to have almost identical functionality to get_earliest_due_date. You should figure out a way to call that to satisfy your need to return 'Not studied yet' here.